core/src/main/java/org/infinispan/marshall/persistence/impl/UserMarshallerWhiteList.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.marshall.persistence.impl;
+
+import java.lang.invoke.SerializedLambda;
+
+import org.infinispan.commons.configuration.ClassWhiteList;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.xsite.BackupSender;
+import org.infinispan.xsite.XSiteAdminCommand;
+
+/**
+ * A white list for Infinispan internal java classes which do not have a {@link org.infinispan.commons.marshall.Externalizer}
+ * instance defined.
+ *
+ * @author Ryan Emerson
+ * @since 10.0
+ */
+class UserMarshallerWhiteList {
+   static void addInternalClassesToWhiteList(ClassWhiteList list) {
+      // Infinispan Enums with no Externalizer
```

Comment:
I am surprised we don't have built in marshalling for Enums in our marshaller? It is just a simple name and ordinal to define any enum. We shouldn't have to add it to a whitelist in that case.

Reply:
We do now :smile: